app/javascript/data/advancedFilterOptions.js

```diff
@@ -0,0 +1,266 @@
+import supportedLanguages from './supportedLanguages.json';
+
+export const advancedFilterOptions = [
```

Comment:
@holmesie would it be better to move these config type things (the advanced filter config and `supportedLanguages`) to the back end and pass them to the front end as React props? This is going to enable the backend to perform validation as well (which is going to be needed for this ticket: https://tracker.codev.mitre.org/browse/SARAALERT-1344)

Reply:
Yeah, I suppose if we're going to need that anyways, we should just do it with this.

Reply:
I've already done a good amount of my work for 1055 off of this, so I think it would be best to create a separate ticket for this one because it includes other things out of the scope of the advanced filter

Reply:
That's fine, this will make it easier to get to that point anyways. Since we want to get this in faster, I'm fine with doing that in a followup ticket.

Reply:
I don't think it would take too much work to move these constants to the back end (just copy and paste to a ruby file and pass it as a prop instead of importing this file). I would be more than happy to help with that if you would like, and I don't think it would be more than changing a couple of lines (for referencing these options) and copying and pasting the multi-line options already added to this file to the one created in the back end. This would also help prevent conflicts with David's work earlier on.

Reply:
upon second thought, this is not too important and we can deal with it later